app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
Please revert this change. This changes changes behavior. For now we can declare strict types =1 only for new files

Reply:
@ihor-sviziev in another PR of mine, the reviewer requested to enable strict types on an existing class: https://github.com/magento/magento2/pull/16571#issuecomment-403402670

Now I am confused :(

Reply:
Ok. I will discuss it with Maintainers and we'll decide what to do in such cases. For now no need to change strict types declaration and arguments for methods, but need to fix optional parameter in constructor

Reply:
After discussion - we decided that we can't use strict types for existing classes as it's not backward compatible change. The same for method arguments and return types in methods that already exists.
https://github.com/magento/magento2/pull/16571#issuecomment-412021296

Please adjust your code. 
Thx

Reply:
@ihor-sviziev  thank you for discussing it, at least it is now clear for me :)